src/components/form-elements/draft-js-mention-selector/DraftJSMentionSelectorCore.js

```diff
@@ -285,6 +301,7 @@ class DraftJSMentionSelector extends React.Component<Props, State> {
                         : []}
                 </SelectorDropdown>
                 {showMentionStartState ? <MentionStartState message={startMentionMessage} /> : null}
+                {announceActiveMention ? <MentionUsersFound /> : null}
```

Comment:
Seems like these could be simplified to `announceActiveMention && ...JSX...` rather than the ternary operator

Reply:
Should `contacts` be passed here?